public/app/features/alerting/unified/Silences.test.tsx

```diff
@@ -312,7 +311,7 @@ describe('Silence edit', () => {
         expect(mocks.api.createOrUpdateSilence).toHaveBeenCalledWith(
           'grafana',
           expect.objectContaining({
-            comment: `created ${now}`,
+            comment: expect.stringMatching(/created (\d{4})-(\d{2})-(\d{2}) (\d{2}):(\d{2})/),
```

Comment:
Could we mock `dateTime().format` instead of applying the regex here to make sure the dates match? No big deal though.

Reply:
So I've tried mocking the `Date.now` constructor and there's a few things to note
1. It's called _a lot_ so might introduce some test weirdness
2. I might be intruding on the "testing implementation details" territory here

Ideally we'd test the `getDefaultFormValues` function that is used in the `SilencesEditor` but I can live with this RegExp for now

Reply:
Thanks for checking, sounds good to me to use the RegExp then 👍